activiti-cloud-services-org/activiti-cloud-services-org-api/src/main/java/org/activiti/cloud/organization/api/ModelType.java

```diff
@@ -22,4 +22,6 @@
 public interface ModelType {
 
     String getName();
+
+    String getZipFolderName();
```

Comment:
I'm wondering if we should not remove `Zip` from the method name -> `getFolderName`. In the case we use it for the extracted structure as well.

Reply:
Agree, let me change it.

Reply:
Done